BullsAndCows/BullsAndCowsGame.cs

```diff
@@ -1,20 +1,66 @@
 ﻿using System;
+using System.Linq;
+using System.Text.RegularExpressions;
 
 namespace BullsAndCows
 {
     public class BullsAndCowsGame
     {
         private readonly SecretGenerator secretGenerator;
+        private readonly string secret = string.Empty;
+        private int gameCount;
         public BullsAndCowsGame(SecretGenerator secretGenerator)
         {
             this.secretGenerator = secretGenerator;
+            this.secret = this.secretGenerator.GenerateSecret();
+            gameCount = 1;
```

Comment:
should the assignment in constructor? assign in property?

Reply:
aggreed